ui/src/components/flows/FlowPlayground.vue

```diff
@@ -23,6 +23,26 @@
                             v-if="executionsStore.execution"
                             :execution="executionsStore.execution"
                         />
+                        <el-button
+                            :icon="CloseIcon"
+                            link
+                            class="tab-icon"
+                            @click="playgroundStore.enabled = false"
+                            :title="t('close')"
+                        />
+                        <el-dropdown trigger="click" placement="bottom-end">
+                            <el-button :icon="DotsVertical" link class="tab-icon" />
+                            <template #dropdown>
+                                <el-dropdown-menu class="m-2">
+                                    <el-dropdown-item @click="playgroundStore.clearExecutions()">
+                                        <span class="small-text">{{ t('clear') }} {{ t('history').toLowerCase() }}</span>
```

Comment:
Can you create a separate constant for this. Ideally it woudl be a subkey in the playground object?

This will pose a problem in any language not exactly structured as English: Verb can be after or before the noun, sometimes its is a single word or more.

```suggestion
                                        <span class="small-text">{{ t('playground.clear_history') }}</span>
```